test/README.md

```diff
@@ -10,15 +10,13 @@ common usage behaviours work on real Linux distros
 To run locally all you need is:
 
 1. Rust nightly
-2. `libelf`
-3. A checkout of [libbpf](https://github.com/libbpf/libbpf)
-4. `cargo install bpf-linker`
-5. `bpftool`
+1. `libelf`
+1. `cargo install bpf-linker`
+1. `bpftool`
```

Comment:
There's an issue tracking the removal of this requirement, can we add a TODO here referencing it?

Reply:
Done.